rfc/0076-subdoc-replica-reads.md

```diff
@@ -0,0 +1,179 @@
+# Meta
+
+| Field          | Value                      |
+|----------------|----------------------------|
+| RFC Name       | Sub-Document Replica Reads |
+| RFC ID         | 76                         |
+| Start Date     | 2023-06-20                 |
+| Owner          | Charles Dixon              |
+| Current Status | DRAFT                      |
+| Revision       | #1                         |
+
+# Summary
+[MB-23162](https://issues.couchbase.com/browse/MB-23162) adds support for performing subdocument read operations on replica servers.
+Replica read operations are performed against replica servers with a new ReplicaRead protocol flag.
+it also includes a new, informational, HELLO flag to be negotiated.
+
+This RFC specifies the SDK work needed to support this feature.
+All SDKs are required to implement this feature.
+
+# Motivation
+SDKs require a way to expose this new ReplicaRead feature in a way that is idiomatic to the SDK API.
+
+# Changes
+
+1. Add two new functions to the `Collection` interface - `LookupInAnyReplica` and `LookupInAllReplicas`.
+2. Add a new `LookupInReplicaResult` result type.
+3. Add two new options types - `LookupInAnyReplicaOptions` and `LookupInAllReplicasOptions`.
+4. Support the new REPLICA_READ (0x20) HELLO flag.
```

Comment:
it is `SubdocReplicaRead = 0x1c`

https://github.com/couchbase/kv_engine/blob/7860a71e4dec0ce24a05e54cfa0dc4c698d62fc1/include/mcbp/protocol/feature.h#L132-L134

Reply:
Good spot, I fixed the codes later in the file but missed these (I confused myself at one point).